Add a Python example of cognitive rest-api

inerplat opened the pull request:
I have written the example code for Python 3.x

This code requested the api using the post of the request package, and did not include the header and processing associated with the 'chunked transfer'.

azurecla commented:
Hi __@Saisang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

inerplat committed:
Add a Python example of cognitive rest-api

I have written the example code for Python 3.x
This code requested the api using the post of the request package, and did not include the header and the processing associated with the chunked transfer.

inerplat committed:
Add python example getting an authorization token

inerplat committed:
Add python requests example by authorization token
